app/views/posts/show.html.erb

```diff
@@ -1 +1,36 @@
-A post details with a list of comments
\ No newline at end of file
+<div class="user-wrapper">
+    <div class="user-container">
+        <div class="user-list">
+            <span class="photo">
+                <%= image_tag(@post.user.photo, size: "100x100" ) %>
+            </span>
+            <span class="user-info">
+                <span>
+                    <%= link_to @post.user.name, user_path(user_id: @post.user.id, format: :html) %>
+                </span>
+                <span>
+                    number of posts: <%= @post.user.posts_counter ? @post.user.posts_counter : 0 %>
+                </span>
+            </span>
+        </div>
+        <ul class="posts">
+            <li>
+                <span>
+                    <%= link_to "Post #: #{@post.id}" , user_post_path(user_id: @post.user.id, post_id: @post.id) %>
+                    by <%=@post.user.name%>
+                </span>
+                <span>
+                    <%= @post.text %>
+                </span>
+                <span>comments: <%= @post.comments.count %>&nbsp; likes: <%= @post.likes.count %></span>
+            </li>
+        </ul>
+        <ul class="comments">
+            <%@post.comments.each do |comment|%>
+                <li>
+                    <%=comment.user.name%> : <%=comment.text%>
+                </li>
+                <%end%>
+        </ul>
```

Comment:
- [ ] When you are on the `posts` page a post should just display only 5 recent comments and it should display all comments when you visit one specific page please address that